kratos/containers/pointer_vector_set.h

```diff
@@ -237,6 +243,12 @@ class PointerVectorSet
     {
         return const_iterator( mData.end() );
     }
+
+    const_iterator cend()
```

Comment:
shouldn't those be `const`
Or add a const overload?

Reply:
But in that case you could use the `const` overload of the standard `begin` & `end` no?

Reply:
no

imagine having a fct that takes the PointerVectorSet as `const` and calls the `cbegin`, then without the const overload it will not compile

IMO having the const overloads for most fcts is part of const correctness

Reply:
Agree. Lets add it for the sake of completeness.

Reply:
The idea of this is having a const iterator from a non const pointer and making const will prevent it. 

About adding the const one I think it maybe useful. (It doesn't needed but can be used when you are not sure if the pointer is const like when it arrives by template argument)

Reply:
Done.